teachopencadd/talktorials/018_md_simulation/talktorial.ipynb

```diff
@@ -0,0 +1,1155 @@
+{
```

Comment:
<blockquote>Execute the following code cell, if on Google Colab.</blockquote><p>Execute the following code cell, if you are working on <a href="https://colab.research.google.com/notebooks/intro.ipynb" target="_blank">Google Colab</a>.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>